entities/src/models.rs

```diff
@@ -73,6 +73,17 @@ pub struct AssetIndex {
     pub fungible_asset_balance: Option<u64>,
 }
 
+/// FungibleAssetIndex is the struct that is stored in the postgres, and is used to query the fungible asset pubkeys.
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Default)]
+pub struct FungibleAssetIndex {
```

Comment:
Following fungible assets data structure, this structure could look like
```
pub struct FungibleAssetIndex {
    pub pubkey: Pubkey,
    pub owner: Pubkey,
    pub slot_updated: i64,
    pub fungible_asset_mint: Pubkey,
    pub fungible_asset_balance: u64,
}
```
Can you please review the changes to see if we can change data structure in such way?

Reply:
Should be fixed